zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -0,0 +1,152 @@
+zipkin:
```

Comment:
Not aware of any yaml tricks (would love them for my kubernetes config!).

But with spring boot, profiles can be used for sharing.

When using the spring boot naming schemes, if you have

```application-zipkin-shared.yml```

Then in ```zipkin-server-aws.yml``` you can add

```spring.profiles.include: zipkin-shared```

To include all the variables.